src/layouts/partials/styles.html

```diff
@@ -3,7 +3,8 @@
 {{- $cssOpts      := (dict "targetPath" $cssTarget "outputStyle" "compressed") }}
 {{- $css := resources.Get $sass | toCSS $cssOpts | minify | fingerprint }}
 
-<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto+Slab:300,700|Roboto:300,700,300italic|Inconsolata:400,700">
+<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto+Slab:300,700|Roboto:300,500,700,300italic|Inconsolata:400,700">
 <link rel="stylesheet" type="text/css" href="https://s.giantswarm.io/giantswarmio/0.10.18/1/bootstrap.min.css">
+<link rel="stylesheet" type="text/css" href="https://cdn2.hubspot.net/hubfs/273774/fontawesome/v5/latest.css">
```

Comment:
Necessary for all the new icons. But since this is shared between the website and the docs, it means that caching helps us, so the impact is not that big.